acceptance_tests/features/environment.py

```diff
@@ -8,8 +8,8 @@
 from acceptance_tests.features.pages import sign_out_internal
 from acceptance_tests.features.steps import authentication
 from config import Config
-from controllers import collection_exercise_controller, database_controller, sample_controller
-from controllers import party_controller, django_oauth_controller, case_controller
+from controllers import (collection_exercise_controller, database_controller, sample_controller,
```

Comment:
Just a quick note here about how we do imports where they end up exceeding line length. This is my preferred method because it's closed to what we do elsewhere in our code. There are some other places where we use / at the end of a line. Either way is good but might be nice to keep things consistent